Sources/Apodini/Semantic Model Builder/InterfaceExporter/HandledRequest.swift

```diff
@@ -0,0 +1,106 @@
+//
+// Created by Andreas Bauer on 22.01.21.
+//
+
+import Foundation
+
+/// A HandledRequest describes the outcome of a `ConnectionContext.handle(...)`.
+public struct HandledRequest: Encodable {
```

Comment:
This is essential a `AnyEncodable` (wrapping the Encodable response) with containing additional context of the handled request (specifically path parameters values and resolvers for relationships).
As its similarly structured to `AnyEncodable`, it can be transparently used by all exporters even if they don't care about relationships.
My initial thought was to name it like `Response` which obviously collided with our existing `Response` enum.
I'm not sure about `ResponseFormatter` as its not only about formatting, but the thing also holds the response itself 🤔

Reply:
nit: I'm also not super happy about the name here. I understand where it comes from though

Reply:
Any other suggestions maybe?

Reply:
As discussed in the meeting: `EnrichedContent` 🚀

Reply:
As of the work meeting, we decided for `EnrichedResponse`